src/store/actions/common/watchListFilter.ts

```diff
@@ -0,0 +1,5 @@
+export const TOGGLE_WATCHED_ONLY = 'TOGGLE_WATCHED_ONLY';
+
+export const toggleWatchedOnly = () => ({
+    type: TOGGLE_WATCHED_ONLY,
```

Comment:
TOGGLE_WATCHED_ONLY - nie do końca mówi o co chodzi za miesiąc możemy mieć problem, 
może TOGGLE_WATCHED_LIST_ONLY albo TOGGLE_WATCHED_LIST i zmiana na to wszędzie